packages/orbit-components/src/Pagination/index.tsx

```diff
@@ -21,6 +19,9 @@ const handlePageChange = (onPageChange, pageCount) => nextPageIndex => {
 
 const Pagination = ({
   pageCount,
+  labelPrev,
+  labelNext,
+  labelProgress,
```

Comment:
Should we have default values for these? Like in the other components

Reply:
here is probably makes sense force them to pass it, they are not optional, plus it needs the translation key for progress, we can't have a default value for that

Reply:
Yes, I was only referring to Next and Prev. But if they are required we don't need default value 👍